src/loaders/compute_tools/busco/versions.yaml

```diff
@@ -0,0 +1,12 @@
+# This tool serves a distinct purpose separate from collection tools; instead, it is suited for CDM work.
+# Therefore, the parser program is not compatible with data generated by this tool.
+
+versions:
+  - version: 0.1.0
+    date: 2024-08-22
+    notes: |
+      - initial BUSCO implementation
+    reference_db_version: odb10
```

Comment:
Remind me, ref data is installed manually right?

Reply:
yea.

Reply:
:+1: